src/backend/app/modules/CustomBulkRegistration/Ecosys/Services/AppliancePersonService.php

```diff
@@ -5,7 +5,7 @@
 use App\Models\AssetPerson;
```

Comment:
We can remove this file.

Reply:
This service is currently being referenced in  `CustomerDatabaseParser` [`src/backend/app/modules/CustomBulkRegistration/Ecosys/CustomerDatabaseParser.php`](https://github.com/EnAccess/micropowermanager/blob/3efe4ae5ce27b699ad65b4ff035997284bc80d16/src/backend/app/modules/CustomBulkRegistration/Ecosys/CustomerDatabaseParser.php#L117)  ) 

Adding this change increases the scope of the PR, can you describe what needs to be done to properly remove this (i.e as a  bullet list)? so I can create a separate ticket just for it.

Reply:
I mean we can remove /modules/CustomBulkRegistration folder.

Reply:
Alright, creating a separate issue specifically for this should be fine then.

Reply:
Issue #529 has been reference